### PR TITLE
v0.14.7 Add Goerli [BAC-2888]

### PR DESCRIPTION
**Pull request by jiajames**

### CODE REVIEW

File: __tests__/StarkwareLib.test.ts [hunk 1]
```diff
@@ -12,7 +13,7 @@ describe('StarwareLib#initiate', () => {
   const dummyProvider = new DummyProvider() as unknown as Provider;
 
   it('Successfully initiates StarkwareLib on Ropsten', async () => {
-    const starkwareLib = new StarkwareLib(dummyProvider, Networks.ROPSTEN);
+    const starkwareLib = new StarkwareLib(dummyProvider, Networks.GOERLI);
```
Comment (eyqs) [inline __tests__/StarkwareLib.test.ts:16]: Successfully initiates StarkwareLib on Goerli

File: __tests__/StarkwareLib.test.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 import { provider as Provider } from 'web3-core';
 
+import usdcAbi from '../src/contracts/usdc-abi.json';
 import { StarkwareLib } from '../src/index';
-import { DummyProvider } from './helpers/DummyProvider';
 import { Networks } from '../src/types';
-import Web3 from 'web3';
+import { DummyProvider } from './helpers/DummyProvider';
 
 // async ropsten calls can take >10k MS
```
Comment (eyqs) [inline __tests__/StarkwareLib.test.ts:9]: async goerli

Comment (eyqs) [reply __tests__/StarkwareLib.test.ts:9]: can remove this since nsb default is 30s

File: src/lib/Constants.ts [hunk 3]
```diff
@@ -15,22 +16,19 @@ export const INTEGERS = {
...
-  USDC_ADDRESS_MAINNET: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
-  USDC_EXCHANGE_ADDRESS_ROPSTEN: '0x144b40e133495811e1d5fd64ab73ed40186ecb17',
-  USDC_EXCHANGE_ADDRESS_MAINNET: '0x032b17633c956c10845643f0bf9ea7c16a3cfb62',
-};
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
 export const USDC_ADDRESSES: {[networkId: number]: string } = {
-  [Networks.ROPSTEN]: ADDRESSES.DYDX_USDC_ADDRESS_ROPSTEN,
-  [Networks.MAINNET]: ADDRESSES.USDC_ADDRESS_MAINNET,
+  [Networks.MAINNET]: usdcAbi.networks[1].address,
```
Comment (eyqs) [inline src/lib/Constants.ts:25]: could index into these using `Networks.MAINNET` instead of 1 3 5

**Review by eyqs — APPROVED** [review]
LGTM

File: __tests__/StarkwareLib.test.ts (hunk 1 above)
Comment (jiajames) [reply __tests__/StarkwareLib.test.ts:13]: done

File: __tests__/StarkwareLib.test.ts (hunk 2 above)
Comment (jiajames) [reply __tests__/StarkwareLib.test.ts]: done

File: src/lib/Constants.ts (hunk 3 above)
Comment (jiajames) [reply src/lib/Constants.ts:25]: didnt seem to work for some reason

**Pull request merged**